src/pages/docs/sdk/api-parameters/url.md

```diff
@@ -34,8 +34,8 @@ export default () => (
 
 It should be reachable by the service. For example, if you want to extract content behind a login panel, the URL provided should contain the authentication step as part of the query string.
 
-If the URL provided has with query strings, it should be properly escaped in order to not interfere with the rest of the API Parameters.
+If the URL provided has query strings, it should be properly escaped in order to not interfere with the rest of the API Parameters.
 
-Also, the protocol matters: If the target URL have relative URLs inside (e.g., images or videos), then the URL provided will be used to resolved relatives URLs into absolute.
+Also, the protocol matters: If the target URL has relative URLs inside (e.g. images or videos), then the URL provided will be used to resolve relative URLs into absolute.
```

Comment:
`e.g.,` is the standard Chicago style we use for referencing examples https://www.chicagomanualofstyle.org/qanda/data/faq/topics/Abbreviations/faq0047.html